### DISCUSSION TITLE
Chapter 04. 데이터 모델링

### DISCUSSION THREAD

**Discussion by new-pow** [CodeSquad-2023-BE-Study/DB-Study#5] [완료]
책을 읽으면서 궁금한 점을 남겨주세요.

**Comment by crtEvent** [comment]
데이터 모델 = 데이터 모델링의 결과를 표현하는 도구 = E-R Model, 계층 데이터 모델 등

도구라고 하면 포토샵, 오토캐드 같은 프로그램 툴이 생각나는데, E-R Model 같이 개념을 표현하는 방식? 도 도구라고 하는 게 저는 어색했어요

**Comment by new-pow** [comment]
발표 주제로 NoSQL의 데이터 모델링 조금 공부해갈게요!

**Comment by new-pow** [comment]
직접 DB 모델링을 해보고 본격적으로 공부해보면 좋을거 같다는 생각을 했습니다.

**Comment by jinny-l** [comment]
호눅스가 EERD랑 ERD의 차이에 대해서도 공부해보면 좋다고 하셨는데
구글링하면 잘 안나오네요

**Reply by crtEvent** [reply]
![image](https://github.com/CodeSquad-2023-BE-Study/DB-Study/assets/86359180/9e8b926b-c65f-4bf7-bac1-493809755445)
이게 ERD

**Reply by new-pow** [reply]
![image](https://github.com/CodeSquad-2023-BE-Study/DB-Study/assets/103120173/5ac6da65-9152-4758-9fa6-aa760269a3f5)

![image](https://github.com/CodeSquad-2023-BE-Study/DB-Study/assets/103120173/7ec6d0f3-5922-4deb-882d-0719ffe929e1)

EERD

EER Diagram, also abbreviated as Enhanced Entity-relationship diagram

**Reply by crtEvent** [reply]
![EERD-of-Telemedicine-Web-Service](https://github.com/CodeSquad-2023-BE-Study/DB-Study/assets/86359180/8b005c09-21f1-4569-80ae-8a6061694d6b)
이게 Relation Table Diagram??

그런데 말입니다. 이걸 EERD라고 하는 글들도 있어요;;;

**Reply by new-pow** [reply]
https://dev.mysql.com/doc/workbench/en/wb-creating-eer-diagram.html

EER 다이어그램 MySQL Workbench Manual 링크입니다.
